dev/NavigationView/NavigationView.cpp

```diff
@@ -215,45 +265,43 @@ void NavigationView::OnApplyTemplate()
 
     m_topNavGrid.set(GetTemplateChildT<winrt::Grid>(c_topNavGrid, controlProtected));
 
+    UpdateNavigationViewItemsFactory();
```

Comment:
Can we do this during MenuItemTemplate/MenuItemTemplateSelector property changed ?